components/supervisor/frontend/src/index.ts

```diff
@@ -262,6 +267,11 @@ LoadingFrame.load().then(async (loading) => {
             });
         }
         const supervisorServiceClient = SupervisorServiceClient.get();
+        if (debugWorkspace) {
+            supervisorServiceClient.supervisorWillShutdown.then(() => {
```

Comment:
(off-topic) just thinking, but with this event, we won't need anymore to know the status phase, i.e

- when supervisorReady then show IDE instead of the loading screen
- when supervisorWIllShutdown show the loading screen instead of IDE

@mustard-mh @jeanp413 ?